quickstarts/calico/config.yml

```diff
@@ -0,0 +1,48 @@
+id:
+slug: calico newrelic
+title: Calico Newrelic
```

Comment:
Same comment here about including the `Newrelic` portion, I'm not sure we need it.

Reply:
Fixed